src/webview/mod.rs

```diff
@@ -386,6 +393,17 @@ impl<'a> WebViewBuilder<'a> {
     self
   }
 
+  /// Configures the given URL to be allowed to load with a self-signed certificate.
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **macOS / Windows / Linux**: Not implemented.
```

Comment:
mac should also get the support I assume?

Reply:
Fixed.